src/js/player.js

```diff
@@ -513,6 +516,8 @@ class Player extends Component {
 
     if (this.tech_) {
       this.tech_.dispose();
+      this.isPosterFromTech_ = false;
```

Comment:
why only set and remove these inside of this if statement? Maybe we just want to do it unconditionally?

Reply:
I didn't want to be greedy and just wanted to limit the effects of my changes. But I suppose they can unconditionally be set.

I believe a `posterchange` event and an update of the `PosterImage` component would be useless here too. Do you concur?

Reply:
Yup, no need to do that during cleanup.